index.html

```diff
@@ -93,10 +117,8 @@
         </a>
 
         <div class="user-greet">
-          <a href="https://my.realdevsquad.com/">
-            <div class="user-greet-msg">Hello, User!</div>
-            <img class="user-profile-pic" />
-          </a>
```

Comment:
why have you removed the anchor tag from here?

Reply:
That is because we do not want to open that link when we click the user greet. Now on clicking user greet you will get a dropdown that contains an option for signout and `my profile` when you click on my profile you can go to the url specified in the above anchor tag.

Reply:
okay